ramalama/model.py

```diff
@@ -252,33 +250,19 @@ def setup_container(self, args):
 
     def gpu_args(self, args, runner=False):
         gpu_args = []
-        machine = platform.machine()
-        if (
-            os.getenv("HIP_VISIBLE_DEVICES")
-            or os.getenv("ASAHI_VISIBLE_DEVICES")
-            or os.getenv("CUDA_VISIBLE_DEVICES")
-            or os.getenv("INTEL_VISIBLE_DEVICES")
-            or os.getenv("ASCEND_VISIBLE_DEVICES")
-            or os.getenv("MUSA_VISIBLE_DEVICES")
-            or (
-                # linux and macOS report aarch64 (linux), arm64 (macOS)
-                ramalama.common.podman_machine_accel
-                or (machine == "aarch64" and os.path.exists("/dev/dri"))
-            )
-        ):
-            if args.ngl < 0:
-                args.ngl = 999
+        if args.ngl < 0:
+            args.ngl = 999
 
-            if runner:
-                gpu_args += ["--ngl"]  # double dash
-            else:
-                gpu_args += ["-ngl"]  # single dash
+        if runner:
+            gpu_args += ["--ngl"]  # double dash
```

Comment:
We must remember to clean this up in a follow on PR, this runner variable isn't required anymore

Reply:
Why not remove it now?